environment/scripts/setup_data.py

```diff
@@ -191,35 +248,250 @@ def download_multiplier_banchereau_mcp_neutrophils(**kwargs):
     )
 
 
-def download_multiplier_recount2_model(**kwargs):
+def download_crispr_lipids_gene_sets_file(**kwargs):
+    output_file = conf.CRISPR["LIPIDS_GENE_SETS_FILE"]
+    curl(
+        "https://upenn.box.com/shared/static/amiu6epztbuqjoad7eq9e50fpfdzdrvt.csv",
+        output_file,
+        "987eeef1987421b596988eba92e6305f",
+        logger=logger,
+    )
+
+
+def download_pharmacotherapydb_indications(**kwargs):
+    output_file = conf.PHARMACOTHERAPYDB["INDICATIONS_FILE"]
+    output_file.parent.mkdir(exist_ok=True, parents=True)
+    curl(
+        "https://ndownloader.figshare.com/files/4823950",
+        output_file,
+        "33585132777601dedd3bed35caf718e2",
+        logger=logger,
+    )
+
+
+def download_lincs_consensus_signatures(**kwargs):
+    output_file = conf.LINCS["CONSENSUS_SIGNATURES_FILE"]
+    output_file.parent.mkdir(exist_ok=True, parents=True)
+    curl(
+        "https://ndownloader.figshare.com/files/4797607",
+        output_file,
+        "891e257037adc15212405af461ffbfd6",
+        logger=logger,
+    )
+
+
+def download_precomputed_gene_correlations(**kwargs):
+    output_file = conf.PHENOMEXCAN["LD_BLOCKS"]["GENE_NAMES_CORR_AVG"]
+    output_file.parent.mkdir(exist_ok=True, parents=True)
+    curl(
+        "https://upenn.box.com/shared/static/xopvxtlj44t4tswz3zzz8uxr6z7dzk0w.pkl",
+        output_file,
+        "cb2f7cabc1aa399a80699076b162a261",
+        logger=logger,
+    )
+
+
+def download_snps_covariance(**kwargs):
+    output_file = conf.PHENOMEXCAN["LD_BLOCKS"]["SNPS_COVARIANCE_FILE"]
+    output_file.parent.mkdir(exist_ok=True, parents=True)
+    curl(
+        "https://upenn.box.com/shared/static/cm9my5bo8jw6nawsxy5y6z0ceh9wnecv.h5",
+        output_file,
+        "bac23b32561f49f1683c421965a4d237",
+        logger=logger,
+    )
+
+
+def download_predixcan_mashr_prediction_models(**kwargs):
+    output_folder = conf.PHENOMEXCAN["PREDICTION_MODELS"]["MASHR"]
+    if output_folder.exists():
+        logger.warning(
+            f"Output directory already exists ({output_folder}). This might cause issues, consider removing it first"
+        )
+        return
+
+    output_folder.parent.mkdir(exist_ok=True, parents=True)
+
+    output_tar_file = Path(
+        conf.PHENOMEXCAN["PREDICTION_MODELS"]["BASE_DIR"], "mashr_eqtl.tar"
+    ).resolve()
+    output_tar_file_md5 = "87f3470bf2676043c748b684fb35fa7d"
+
+    if not Path(output_tar_file).exists() or not md5_matches(
+        output_tar_file_md5, output_tar_file
+    ):
+        # download
+        curl(
+            "https://zenodo.org/record/3518299/files/mashr_eqtl.tar?download=1",
+            output_tar_file,
+            output_tar_file_md5,
+            logger=logger,
+        )
+
+    # uncompress file
+    import tarfile
```

Comment:
Just to check, is there a reason to have this here, or should it be at the top? It seems like several functions require it!